### PR TITLE
Use jsonpickle in django contrib

### PR DESCRIPTION
**Pull request by chripede**
This change will try to import jsonpickle and use that to pickle and unpickle the token stored in database.

Discussed in #672

### CODE REVIEW

File: oauth2client/contrib/django_util/models.py [hunk 1]
```diff
@@ -16,6 +16,12 @@
 
 import base64
 import pickle
+import sys
+
+try:
+    import jsonpickle
+except ImportError:
+    pass
```
Comment (waprin) [inline oauth2client/contrib/django_util/models.py]: I think I already made jsonpickle a dependency of this contrib package? Have you tested it with regular pickle? Think there are some subtle issues...maybe best to just require jsonpickle.

Comment (chripede) [reply oauth2client/contrib/django_util/models.py]: jsonpickle is not installed with the package, but I'd prefer to do that if it's okay.

Comment (waprin) [reply oauth2client/contrib/django_util/models.py]: It's used in `contrib.django_util.views.py`. It's not installed with oauth2client, but necessary to use the django contrib.

What we really want is something alone the lines of making the Django contrib package an optional dependency, which brings in jsonpickle with it. For the purpose of this PR I would just assume it's installed.

Comment (chripede) [reply oauth2client/contrib/django_util/models.py]: That's great. Cleaned up the code.

**Comment by waprin** [comment]
Looks like you have some flake errors

**Comment by chripede** [comment]
Not really sure why, as I never touched the files that fails. I've fixed them and everything seems fine now

File: oauth2client/contrib/devshell.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ class CommunicationError(Error):
 class NoDevshellServer(Error):
     """Error when no Developer Shell server can be contacted."""
 
+
```
Comment (waprin) [inline oauth2client/contrib/devshell.py:40]: was this needed to pass flake?

File: tests/test_service_account.py [hunk 3]
```diff
@@ -369,6 +369,7 @@ def test_access_token(self, utcnow):
 
         self.assertEqual(credentials.access_token, token2)
 
+
```
Comment (waprin) [inline tests/test_service_account.py:372]: was this needed to pass flake?

**Comment by waprin** [comment]
Ok I see, yeah I don't think we pin the linter so sometimes lint errors pop up underneath us.

**Comment by waprin** [comment]
Thanks for PR.

**Pull request merged**